charts/private-action-runner/templates/deployment.yaml

```diff
@@ -0,0 +1,57 @@
+{{- range $_, $runner := $.Values.runners }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "chart.deploymentName" $runner.name }}
+  namespace: {{ $.Values.common.namespace }}
```

Comment:
I see other charts are using .Release.namespace instead ? 

https://github.com/DataDog/helm-charts/blob/97c1f224a56cb0ff15d26f25571d12de08222740/charts/datadog-operator/templates/deployment.yaml#L5C18-L5C25

Reply:
oh yes, changed that!

Reply:
We need `.Values.common.namespace` since it is a way to reference the namespace value defined under the common section of the values.yaml file. I think we want the user to set this because we want them to create a new namespace for the runner.

Reply:
oh then why do we have `Create the kubernetes namespace kubectl create namespace private-action-runner` in the original instructions and why is there 
`namespace: private-action-runner`
https://github.com/DataDog/experimental/blob/29d4c9a71717b8b29966671eb2c101c9eaf1edff/teams/workflow-automation/private-actions/helm-chart/private-action-runner/values.yaml#L2 
in the config?

Reply:
I don't exactly remember why I put it there
It might be a leftover of when this was not a helm chart and it was simpler to delete the namespace to restart from scratch. Sorry about that 😞 
But I don't think there is a major reason we should keep it like that and I think following the other charts makes more sense.

Reply:
ok sounds good, makes things simpler!